### PR TITLE
Sizing.Random.random()™

### PR DESCRIPTION
**Pull request by chyzman**
also updated wisdom

### CODE REVIEW

File: src/main/java/io/wispforest/owo/util/Wisdom.java [hunk 1]
```diff
@@ -54,7 +54,8 @@ private Wisdom() {}
             "i ate moss and i died",
             "Jello was 126 years old when we added this line",
             "I saw ppl complain that minor updates between big updates will ruin it for modders but I thought modders were very good and they can do what we do in a week so they should be ok with updating their mods to new versions in a day, right? :titantroll:",
-            "that's a CanPickUpLoot baby zombie, the most annoying thing ever. he runs around like crazy and picks up all your shit"
+            "that's a CanPickUpLoot baby zombie, the most annoying thing ever. he runs around like crazy and picks up all your shit",
+            "blod: I think I need to take a book out of your page"
```
Comment (BasiqueEvangelist) [inline src/main/java/io/wispforest/owo/util/Wisdom.java]:
```suggestion
            "blod: I think I need to take a book out of your page",
            "blod: he her"
```

File: src/main/java/io/wispforest/owo/ui/core/Color.java [hunk 2]
```diff
@@ -69,6 +69,15 @@ public static Color ofDye(@NotNull DyeColor dyeColor) {
         return new Color(ColorHelper.Argb.getRed(color), ColorHelper.Argb.getGreen(color), ColorHelper.Argb.getBlue(color));
     }
 
+    /**
+     * Generates a random color
+     * @apiNote Don't tell glisco about this
```
Comment (gliscowo) [inline src/main/java/io/wispforest/owo/ui/core/Color.java:74]: Noted, won't do

File: src/main/java/io/wispforest/owo/ui/core/Sizing.java [hunk 3]
```diff
@@ -90,6 +91,229 @@ public static Sizing expand(int percent) {
...
+         */
+        public static Sizing random(int max) {
+            return random(0, max);
+        }
+
+        /**
+         * Generate a random sizing instance with a value between 0 and 100
+         *
+         * @return A random sizing instance
+         * @apiNote Can crash if put on a component that isn't a container
```
Comment (gliscowo) [inline src/main/java/io/wispforest/owo/ui/core/Sizing.java]: More like, any component that doesn't support content sizing - scroll containers come to mind (at least on one of their axes)

File: src/main/java/io/wispforest/owo/ui/core/Sizing.java [hunk 4]
```diff
@@ -90,6 +91,229 @@ public static Sizing expand(int percent) {
...
+    public static class Random {
+        private static final java.util.Random SIZING_RANDOM = new java.util.Random();
+
+        /**
+         * Generate a random fill sizing instance with a value between {@code min} and {@code max}
+         *
+         * @param min The minimum value
+         * @param max The maximum value
+         * @return A random sizing instance
+         * @author chyzman
```
Comment (gliscowo) [inline src/main/java/io/wispforest/owo/ui/core/Sizing.java]: No need for an author tag on every method if there's already one on the class

**Review by gliscowo — CHANGES_REQUESTED** [review]

File: src/main/java/io/wispforest/owo/ui/core/Color.java (hunk 2 above)
Comment (BasiqueEvangelist) [reply src/main/java/io/wispforest/owo/ui/core/Color.java:74]: hey @gliscowo you might wanna see this

File: src/main/java/io/wispforest/owo/ui/core/Sizing.java [hunk 5]
```diff
@@ -182,11 +388,7 @@ private MergedSizing(Sizing first, Sizing second, float delta) {
 
         @Override
         public int inflate(int space, Function<Sizing, Integer> contentSizeFunction) {
-            return MathHelper.lerp(
-                    this.delta,
-                    this.first.inflate(space, contentSizeFunction),
-                    this.second.inflate(space, contentSizeFunction)
-            );
+            return MathHelper.lerp(this.delta, this.first.inflate(space, contentSizeFunction), this.second.inflate(space, contentSizeFunction));
```
Comment (gliscowo) [inline src/main/java/io/wispforest/owo/ui/core/Sizing.java]: ngl, feels like u pressed ctrl+alt+l a bit too hard

**Review by gliscowo — APPROVED** [review]
Finally, the perfect PR

**Pull request merged**